forest/src/cli/chain_cmd.rs

```diff
@@ -48,6 +69,82 @@ impl ChainCommands {
                 let cid: Cid = cid.parse().unwrap();
                 print_rpc_res_pretty(chain_get_block((CidJson(cid),)).await);
             }
+            Self::Export {
+                tipset,
+                recent_stateroots,
+                skip_old_messages,
+                output_path,
+            } => {
+                let chain_finality = Policy::default().chain_finality;
```

Comment:
shouldn't we take policy from the configuration? otherwise issues may arise in case the correct value is not the default one

Reply:
We should, however I wasn't able to figure out a way to access the current config from the CLI. Specifically within this command. I'll look into this some more, maybe we can pass the config to each command as a refactor?